x-pack/solutions/security/packages/features/src/security/v4_features/kibana_features.ts

```diff
@@ -73,9 +100,33 @@ export const getSecurityV4BaseKibanaFeature = ({
   ),
   privileges: {
     all: {
+      replacedBy: {
+        default: [
+          { feature: SECURITY_FEATURE_ID_V5, privileges: ['all'] },
+          { feature: RULES_FEATURE_ID, privileges: ['all'] },
+        ],
+        minimal: [
+          { feature: SECURITY_FEATURE_ID_V5, privileges: ['minimal_all'] },
+          { feature: RULES_FEATURE_ID, privileges: ['minimal_all'] },
+        ],
+      },
       app: [APP_ID, CLOUD_POSTURE_APP_ID, CLOUD_DEFEND_APP_ID, 'kibana'],
       catalogue: [APP_ID],
-      api: [APP_ID, 'rac', 'lists-all', 'lists-read', 'lists-summary'],
+      api: [
+        APP_ID,
+        'rac',
+        LISTS_API_ALL,
+        LISTS_API_READ,
+        LISTS_API_SUMMARY,
```

Comment:
Was it agreed upon somewhere that value lists are going to be part of the rules feature? My understanding of the PRD was that lists would be moved out as a separate feature in the future. cc: @approksiu If that is still the case, I think value list access should remain a part of the SIEM feature.

Reply:
We had discussed it during one of our syncs and decided that value lists should be moved over along with rules. They are currently only used for rule exceptions. We still do plan to move out value lists, but are on standby to see what the RBAC architecture is for lookup indices.